qiskit_experiments/calibration_management/backend_calibrations.py

```diff
@@ -67,6 +85,19 @@ def __init__(self, backend: Backend):
         for meas, freq in enumerate(backend.defaults().meas_freq_est):
             self.add_parameter_value(freq, self.meas_freq, meas)
 
+        basis_gates = basis_gates or list()
+
+        if library is not None:
+            library_instance = library(library_options)
+
+            # Add the basis gates
+            for gate in basis_gates:
+                self.add_schedule(library_instance[gate])
```

Comment:
I guess we need to raise an error when gate is not supported by the library.

Reply:
Yes, this should do it: 59bde3e